src/store/models/bitcoind.ts

```diff
@@ -8,7 +8,11 @@ import { RootModel } from './';
 
 const { l } = prefixTranslation('store.models.bitcoind');
 
+export const getNetworkBackendId = (node: BitcoinNode) =>
```

Comment:
Nice fix on this one. I guess it's never been a problem since users could only ever take action on the current network being displayed in the UI.

I suggest moving this function into the `utlis/network.ts` file because it's cleaner to have components import utils vs importing from the store models.